src/mscorlib/shared/System/Globalization/TimeSpanParse.cs

```diff
@@ -635,7 +635,7 @@ internal static bool TryParseExactMultiple(ReadOnlySpan<char> input, string[] fo
         /// <summary>Common private Parse method called by both Parse and TryParse.</summary>
         private static bool TryParseTimeSpan(ReadOnlySpan<char> input, TimeSpanStandardStyles style, IFormatProvider formatProvider, ref TimeSpanResult result)
         {
-            input = input.Trim();
```

Comment:
What was wrong with `input.Trim()` ?

Reply:
We have added MemoryExtension methods (Trim/etc) and the call was ambiguous. I will clean up/remove StringSpanHelpers APIs in favor of MemoryExtension after this change goes through.

Reply:
IMHO, it would be better to just delete StringSpanHelpers.Trim in this change instead of going back and forth.

Reply:
There are other APIs from StringSpanHelpers we need to delete (and update uses). Most of the calls specify the class name and hence weren't ambiguous. I wanted to keep that change together (and isolated from this one).

> Cleanup StringSpanHelpers and uses in subsequent PR.

I can do the cleanup change as part of this PR, but it would have to wait till tomorrow. Hope that's OK.

Reply:
Bulk of the cleanup can wait for other PR. I just meant that we can do a little bit of the cleanup here to avoid useless changes.